scripts/gha/setup-training-operator.sh

```diff
@@ -27,7 +27,7 @@ cd manifests/overlays/standalone
 kustomize edit set image kubeflow/training-operator=${TRAINING_CI_IMAGE}
 
 echo "Installing training operator manifests"
-kustomize build . | kubectl apply -f -
+kustomize build . | kubectl apply --server-side=true -f -
```

Comment:
I'm ok with follow-up.

Could you update the installation documentation in the following?
- https://github.com/kubeflow/training-operator?tab=readme-ov-file#installation
- https://www.kubeflow.org/docs/components/training/installation/#installing-the-training-operator

Reply:
I've updated the README. I'll raise a PR in the website repository promptly.

Reply:
Thanks!